openedx/core/djangoapps/credit/api.py

```diff
@@ -164,3 +175,21 @@ def _validate_requirements(requirements):
                 )
             )
     return invalid_requirements
+
+
+def is_credit_course(course_key):
```

Comment:
It looks like this function has been designed to accept either a CourseKey object, or the unicode representation of one.  If so, the docstring should explain that explicitly.